playwright/src/test/java/com/microsoft/playwright/BrowserFromEnv.java

```diff
@@ -0,0 +1,34 @@
+package com.microsoft.playwright;
+
+import com.microsoft.playwright.junit.BrowserFactory;
+
+import static com.microsoft.playwright.Utils.getBrowserChannelFromEnv;
+import static com.microsoft.playwright.Utils.getBrowserNameFromEnv;
+
+public class BrowserFromEnv implements BrowserFactory {
```

Comment:
Do we want to use `HEADLESS` instead so that the API to configure tests and for playwright are the same?

Reply:
`HEADED` is preferred as it defaults to `false` when not specified (rather than headless that is true by default and only if specified as HEADLESS=0 it actually changes the behavior), consistent with --headed in node.js command line and  `HEADED` in .Net. We didn't change the parameter in the library API as it had already been shipped by the time we realized this in the test runner.